cloud: the Point-in-time Restore feature can be diabled when weekly backup is enabled (#21642)
pingcap/docs: tidb-cloud/backup-and-restore.md

        > **Warning**
        >

>         > - When weekly backup is enabled, the Point-in-time Restore feature is enabled by default and cann be disabled.

Reviewer: ```suggestion
        > - When weekly backup is enabled, the Point-in-time Restore feature is enabled by default and you can diable it manually.
```

>         > - When weekly backup is enabled, the Point-in-time Restore feature is enabled by default and cann be disabled.
>         > - If you change the backup cycle from weekly to daily, the Point-in-time Restore feature remains its original setting. You can manually disable it if needed.

Reviewer: ```suggestion
        > - If you change the backup cycle from weekly to daily, the Point-in-time Restore feature remains its original setting. You can manually disable it if needed.
```
